pkgs/tools/text/zimreader/tntnet221.patch

```diff
@@ -0,0 +1,62 @@
+--- a/src/main.cpp	2016-08-17 00:54:46.329502545 +0200
```

Comment:
The PR I've sent to openzim repo has not been merged yet.
The patch is still required.